diff.py

```diff
@@ -1477,6 +1520,20 @@ class ArchSettings:
     instructions_with_address_immediates=MIPS_BRANCH_INSTRUCTIONS.union({"jal", "j"}),
 )
 
+ARM32_SETTINGS = ArchSettings(
+    re_int=re.compile(r"[0-9]+"),
+    re_comment=re.compile(r"(<.*?>|//.*$)"),
+    # GPRs and FP registers: r0..15, pc, lr
```

Comment:
There are also no VFP/NEON registers in the list at the moment.

Reply:
Not very familiar with these. Please let me know if I specified anything incorrectly.

Reply:
The list is not incorrect per se – it's just missing all the floating point registers. However, the comment is misleading because:

* r0..r15 includes r14, which is already mentioned separately as the link register (LR)
* r15 also appears twice (PC), and PC isn't considered a general-purpose register.
* There are no floating-point registers in the description or in the code.

Reply:
Okay, I added in the single-precision floating-point registers and tried to clarify the comment.